eva/executor/seq_scan_executor.py

```diff
@@ -44,8 +44,8 @@ def exec(self, **kwargs) -> Iterator[Batch]:
         for batch in child_executor.exec(**kwargs):
             # apply alias to the batch
             # id, data -> myvideo.id, myvideo.data
-            if self.alias:
-                batch.modify_column_alias(self.alias)
+            # if self.alias:
```

Comment:
Add note explaining why code is commented out.

Reply:
the comments have been removed. The code logic has been updated so the commented block is not needed anymore,